src/Common/src/System/Dynamic/Utils/ExpressionUtils.cs

```diff
@@ -141,35 +141,35 @@ public static void ValidateArgumentCount(MethodBase method, ExpressionType nodeK
             }
         }
 
-        public static Expression ValidateOneArgument(MethodBase method, ExpressionType nodeKind, Expression arg, ParameterInfo pi)
+        public static Expression ValidateOneArgument(MethodBase method, ExpressionType nodeKind, Expression arguments, ParameterInfo pi)
```

Comment:
Understood on the reasoning (it's the same reasoning that applies to some of the others), but the path back to the public surface area wasn't obvious... does this always come from calls to InvocationExpression.Invoke?